File: Programming/C++/00/0008 X2.cpp
#include<bits/stdc++.h>
using namespace std;
int X1,X2,S;

int main ()
{
    // Input X1 and S
    cin >> X1 >> S;

    // Calculate and output X2
    // S = (X1+X2)/2
    // 2S = X1+X2
    // X2 = 2S - X1
    X2 = (2*S) - X1;
    cout << X2;
}

File: Programming/Python/00/0008 X2.py
# Input X1 and S
data = input().split()
X1,S = int(data[0]), int(data[1])

# Calculate and Ouput X2
# S = (X1 + X2)/2
# 2S = X1 + X2
# X2 = 2S - X1
X2 = 2*S - X1
print(X2)